packages/circuit-ui/components/Select/Select.tsx

```diff
@@ -61,9 +61,9 @@ export interface SelectProps extends SelectHTMLAttributes<HTMLSelectElement> {
    */
   invalid?: boolean;
   /**
-   * Currently selected value. Matches the "value" property of
-   * the options objects. If value is falsy, Select will render
-   * the "placeholder" prop as currently selected.
+   * Currently selected value. Matches the "value" property of the options
+   * objects. If value is falsy, Select will render the "placeholder" prop as
+   * currently selected.
```

Comment:
Do you have an editor plugin that does these line breaks automatically?

Reply:
Nope 🙈 just do it manually whenever I'm already tweaking descriptions in a file. A plugin (or prettier rule? 🥺) would be awesome